frontend/src/components/widgets/Multiselect/Multiselect.test.tsx

```diff
@@ -119,6 +119,17 @@ describe("Multiselect widget", () => {
     expect(wrapper.find(UISelect).prop("valueKey")).toBe("value")
   })
 
+  it("should filter based on label, not value", () => {
```

Comment:
Nit: `it("filters based on label, not value", ...` ("should" is redundant & adds clutter, IMO)

Reply:
Agree with this, hadn't considered it.